extension/app/src/components/conversation/ConversationViewer.tsx

```diff
@@ -35,7 +35,9 @@ export function ConversationViewer({
     workspaceId: owner.sId,
   });
 
-  const messages = (conversation?.content || []).flat();
+  const messages = (conversation?.content || []).map(
```

Comment:
Worth a comment to explain that's because each array contains all versions of the message and we want only the last version? 🙏🏻